examples/component/.gitignore

```diff
@@ -1,2 +0,0 @@
-example_component
```

Comment:
Ah, if I recall correctly, I did something that caused it to not compile the example component anymore. Also, it surprisingly compiles much faster without the example component. Additionally, there is still an implementation of an example component in the component test by necessity.